src/components/views/card-list.tsx

```diff
@@ -11,7 +11,7 @@ const CardList = ({items, h3Heading}: {items: NodeUnion[]; h3Heading?: boolean})
         {items
           .filter(item => !!item?.id)
           .map(item => (
-            <li key={item.id} className="mx-auto w-full max-w-[500px]">
+            <li key={item.id} className="mx-auto w-full">
```

Comment:
This (along with a few other 3-column layouts in lists) actually prevents the horizontal card from appearing, unless there’s a case where StanfordPeopleCard is imported but not used in a 3-column layout 🤔